src/components/FileItem/FileItem.tsx

```diff
@@ -11,7 +11,7 @@ export const FileItem = () => {
   const { filename } = useParams();
   const navigate = useNavigate();
 
-  const { data, error, isLoading } = useGetFileByNameQuery(filename!);
+  const { data } = useGetFileByNameQuery(filename || "");
```

Comment:
Bad idea, this can trigger empty requests, add skip property based on filename

Reply:
```
let data;
  if (filename) {
    data = useGetFileByNameQuery(filename).data;
  }
```

Is it ok?

Reply:
Here is another variant

`const { data } = filename
    ? useGetFileByNameQuery(filename)
    : ({ data: "" } as File);`

Reply:
Both of them don't work